tutorials/rnn/ptb/util.py

```diff
@@ -33,7 +33,7 @@ def export_state_tuples(state_tuples, name):
 
 def import_state_tuples(state_tuples, name, num_replicas):
   restored = []
-  for i in range(len(state_tuples) * num_replicas):
+  for i in range(len(state_tuples) * (num_replicas or 1)):
```

Comment:
how about putting assert num_replicas > 0?
I think the caller of import_state_tuples should pass a valid num_replicas based on num of cpu/gpus.

Reply:
As  I explained https://github.com/tensorflow/models/issues/2505#issuecomment-334936514 "on a machine with gpus == 0 it [raises](https://github.com/tensorflow/models/blob/1464a721fef7da67e87b0f1f1a710b1e904e8764/tutorials/rnn/ptb/ptb_word_lm.py#L449) for num_gpus == 1 (the default value). "

Reply:
We should probably redo all the error checking but I believe this will do for now